source/develop/release-management/features/sla/soft-host-to-vm-affinity-support.html.md

```diff
@@ -0,0 +1,110 @@
+---
+title: soft-host-to-vm-affinity-support
+category: feature
+authors: yquinn,msivak
+wiki_category: Feature
+wiki_title: Features/soft-host-to-vm-affinity-support
+wiki_last_updated: 2016-11-23
+---
```

Comment:
The 'wiki' tags are only for old pages. Please remove them.

Also you have missing headers in this format :
title: DiscardAfterDelete
category: feature
authors: ishaby
feature_name: Discard after delete
feature_modules: vdsm
feature_status: Partially implemented in oVirt 4.0